server/usr/share/pulp/templates/directory_index.html

```diff
@@ -0,0 +1,31 @@
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Pulp Repository Index</title>
+</head>
+<body>
+    <h1>Pulp Repository Content</h1>
+    <a href="../">Parent directory</a>
+
+    {% if dirs %}
+        <h2>Directories</h2>
+        <ul>
+            {% for d in dirs %}
+            <li><a href="{{ d }}/">{{ d }}</a></li>
+            {% endfor %}
+        </ul>
+        {% endif %}
+
+        {% if files %}
```

Comment:
Yup. This sorts the content of a repository into directories and files and puts all the directories at the top (so you don't have to go hunting for `repodata` somewhere in 1000s of files). The contents of `repodata` would just be files, though.